environment.yml

```diff
@@ -17,10 +17,11 @@ dependencies:
   - google-cloud-storage
   - azure-storage-blob
   - boto
+  - boto3
   - pymongo
   - docker-compose
   - pymysql
   - pre-commit
-  - setuptools-scm
+  - setuptools-scm==6.4.2
```

Comment:
This pin is necessary to support Python 3.6, right? In that case, I'd suggest removing the Python 3.6 support (in a separate PR).

Reply:
Yep! See #49.